sbt/src/sbt-test/dependency-management/mvn-local/test

```diff
@@ -43,4 +43,5 @@ $ copy-file changes/libDeps.sbt library/build.sbt
 > library/publishM2
 
 # should succeed now that the dependency is there
+$ sleep 1000
```

Comment:
Scripted error was `{line 46} Command failed`, which is `compile`. If compile is failing because it's not finding the right JAR, I figured incrementing 1s could nudge Ivy to look harder. Not sure if this theory holds.